### PR TITLE
RET-2616-fix

### PR DESCRIPTION
**Pull request by ikirsanov**
### JIRA link
https://tools.hmcts.net/jira/browse/RET-2616

### Change description
Bug fixes: 
     Contact the tribunal selected page fixes: 
               - Change the fontsize in njk
               - typo in translations
               - cancel link redirect to citizen hub                   

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/controllers/ContactTheTribunalSelectedController.ts [hunk 1]
```diff
@@ -104,9 +105,8 @@ export default class ContactTheTribunalSelectedController {
     }
     req.session.errors = [];
 
-    const redirectPage = userCase.copyToOtherPartyYesOrNo
-      ? PageUrls.CONTACT_THE_TRIBUNAL_CYA
-      : PageUrls.COPY_TO_OTHER_PARTY;
+    const redirectPage =
+      'witness' === userCase.contactApplicationType ? PageUrls.CONTACT_THE_TRIBUNAL_CYA : PageUrls.COPY_TO_OTHER_PARTY;
```
Comment (marcjudge) [inline src/main/controllers/ContactTheTribunalSelectedController.ts:109]: Use a constant for this string

**Review by marcjudge — CHANGES_REQUESTED** [review]

**Review by paul-peasegood — DISMISSED** [review]

File: src/main/controllers/ContactTheTribunalSelectedController.ts (hunk 1 above)
Comment (ikirsanov) [reply src/main/controllers/ContactTheTribunalSelectedController.ts]: Done

**Review by marcjudge — DISMISSED** [review]

File: src/main/definitions/constants.ts [hunk 2]
```diff
@@ -412,3 +412,7 @@ export const languages = {
   WELSH_URL_PARAMETER: '?lng=cy',
   ENGLISH_URL_PARAMETER: '?lng=en',
 };
+
+export const ContactApplications = {
+  WITNESS: 'witness',
```
Comment (marcjudge) [inline src/main/definitions/constants.ts:417]: Lets rename this as TypeC

Comment (ikirsanov) [reply src/main/definitions/constants.ts]: Done

**Review by marcjudge — APPROVED** [review]

**Review by Itay-Kirsh — APPROVED** [review]

**Pull request merged**